conda/plugins/subcommands/doctor/health_checks.py

```diff
@@ -19,6 +23,10 @@
 
 def display_report_heading(prefix: str) -> None:
     """Displays our report heading."""
+    warnings.warn(
+        "The `display_report_heading` function is pending deprecation and will be removed in a future release. ",
+        PendingDeprecationWarning,
+    )
```

Comment:
Deprecations are done using the `conda.deprecations.deprecated` function decorator.